apps/coscrad-frontend-e2e/src/e2e/shared/seed-dummy-media-item.cy.ts

```diff
@@ -0,0 +1,28 @@
+import { AggregateType, MIMEType } from '@coscrad/api-interfaces';
+
+export const seedDummyMediaItem = ({
```

Comment:
I'd prefer to use the cypress commands chain instead. And do we really need the `api-interfaces` here? The CLI already dynamically validates types, and typically, we're only overriding one or two props.

Reply:
The tooling wasn't built with static type sharing in mind. I'd be worried about circular deps.

Reply:
Ok sounds good, I thought that might be a reach.  Not sure how to implement a cypress command for it so would need your help.

Reply:
No problem. I went ahead and did this refactor.